### PR TITLE
OPENEUROPA-1360: Add starter files from site template.

### PR DESCRIPTION
**Pull request by dxvargas**
### Description

Add starter files from site template.

### Change log

- Added: Starter files from site template.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
+    "name": "openeuropa/oe_editorial",
+    "description": "OpenEuropa Drupal module template.",
+    "type": "drupal-module",
+    "license": "EUPL-1.2",
+    "minimum-stability": "dev",
+    "prefer-stable": true,
+    "require": {
+        "openeuropa/drupal-core-require": "^8.6",
```
Comment (upchuk) [inline composer.json]: Let's remove this please. We can directly use drupal/core

Comment (voidtek) [reply composer.json]: Removing this dependency is not in the scope of this ticket.
We should create a follow-up ticket and include the changes in the template drupal module.

Comment (upchuk) [reply composer.json]: OK, fair enough. Can you please create the followup?

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,62 @@
+{
+    "name": "openeuropa/oe_editorial",
+    "description": "OpenEuropa Drupal module template.",
```
Comment (voidtek) [inline composer.json]: The description should be updated.

Comment (dxvargas) [reply composer.json]: Done.

**Pull request merged**